csharp/examples/OpenStudio.Tests/OpenStudio.Tests.csproj

```diff
@@ -1,22 +1,18 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
-
+    <TargetFrameworks>NET45;NET5</TargetFrameworks>
+    <AppendTargetFrameworkToOutputPath>true</AppendTargetFrameworkToOutputPath>
+    <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
+    <OutputPath>bin/$(Configuration)</OutputPath>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
-    <PackageReference Include="xunit" Version="2.4.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
-    <PackageReference Include="coverlet.collector" Version="1.0.1" />
-    <Reference Include="OpenStudio">
-      <!-- OpenStudio.dll needs to be in the same folder as libopenstudio_csharp.so libopesntudio_model_csharp.so and libopenstudio_translators_csharp.so
-      (or a symlink works too: for me in build folder, I did `cd Products && ln -s ./csharp/Release/netcoreapp3.0/OpenStudio.dll ./OpenStudio.dll` -->
-      <HintPath>/home/julien/Software/Others/OS-build-release/Products/OpenStudio.dll</HintPath>
-    </Reference>
-
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.1" />
+    <PackageReference Include="NUnit" Version="3.13.1" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
+    <PackageReference Include="OpenStudio" Version="3.2.0" />
```

Comment:
This version needs to be parametrized by CMake instead of being hardcoded.

Reply:
This package will be replaced with new build nuget package in the GitHub action. The package reference here is just for developers who doesn't compile the OpenStudio on their machine but just use this example folder for their test. So this ensures this test project doesn't miss any dlls or other files that they need to copy from different locations. It will just work once the new nuget package is published.

![image](https://user-images.githubusercontent.com/9031066/116176900-e59ba500-a6e0-11eb-9766-59ecfbcca3bc.png)

https://github.com/NREL/OpenStudio/pull/4287/files#diff-d4f21227b0c5149044e865e56d2a7b63af2067d3aee7cdd0701a2dc01adf8f34R109-R111